packages/rocketchat-api/server/api.js

```diff
@@ -139,15 +167,22 @@ class API extends Restivus {
 		}
 
 		const version = this._config.version;
-
+		const shouldAddRateLimitToRoute = options.rateLimiterOptions && version;
```

Comment:
Could we have a default reate limite? If not specified the default would be applied, if you wanna disable it you could pass `fasse`